test/mongoose_domain_core_SUITE.erl

```diff
@@ -138,15 +138,13 @@ can_get_outdated_domains(_) ->
     [] = mongoose_domain_core:get_all_outdated(another_dummy_src).
 
 run_for_each_domains(_) ->
-    NumOfDomains = 10,
+    NumOfDomains = 1234,
```

Comment:
Please add a comment so it does not seem to be a magic random number. It took me a while to figure it out and I know why we need it to large (above the batch size), but it's not obvious.